guides/source/configuring.md

```diff
@@ -1032,6 +1032,7 @@ The default behavior is to report all warnings. Warnings to ignore can be specif
   config.active_record.db_warnings_ignore = [
     /Invalid utf8mb4 character string/,
     "An exact warning message",
+    "1062",
```

Comment:
What do you think about adding `# MySQL Error 1062: Duplicate entry` like you have above in the CHANGELOG code example to make this a bit clearer?